mods/xpanes/init.lua

```diff
@@ -8,7 +8,7 @@ local directions = {
 	{x = 1, y = 0, z = 0},
 	{x = 0, y = 0, z = 1},
 	{x = -1, y = 0, z = 0},
-	{x = 0, y = 0, z = -1},
+	{x = 0, y = 0, z = -1}
```

Comment:
This should stay here.  It makes it easier to add more entries and looks more consistent.